values.yaml

```
@@ -16,6 +16,16 @@ serviceAccount:
   name: ""
 
 web:
+  livenessProbe:
+    enabled: true
+  readinessProbe:
+    enabled: true
+  autoscaling:
+    enabled: true
+    minReplicas: 1
+    maxReplicas: 5
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 60
```

```suggestion
    targetMemoryUtilizationPercentage: 80
```

As this container boots really quick and with the adjusted timeouts as suggested above we can slightly delay the autoscaling to happen only at 80% as default values for this chart.